### ISSUE TITLE
Shift+Del dialog focus on checkbox instead of button

### ISSUE THREAD

**Issue by Scrama** [linuxmint/xviewer#99]
```
 * Xviewer version (xviewer 2.4.3)
 * Distribution - (Mint 19.3 Mate)
```

In shift+delete dialog default focus is on checkbox "don't ask..." instead of "Delete" button

**Steps to reproduce**
Open image
Press shift+del
Press enter to accept warning and delete file

**Expected behaviour**
file removed
**Actual behaviour**
checkbox state changed, me frustrated )

**Other information**
Actually it is not a bug, but fails usability for me every time when I'm filtering new photos

**Comment by programmer-ceds** [comment]
I have just tested a simple change to give the delete confirmation button focus when the dialog appears. Two issues though.

Firstly this change would nake the behaviour of the program more 'dangerous' - does anyone else have any feelings about this?

Secondly I already have one pull request that is being blocked due to a dependency on xapps (any any other PRs would hit the same problem). Once this has been resolved I will submit the change for this issue if others think that it should be changed in this way.

Nemo is set to operate in the way that you request although there are other applications that require the extra step of clicking on the delete button.
